src/components/QRShare/types.ts

```diff
@@ -49,6 +49,21 @@ type QRShareProps = {
      * The size ratio of margin around logo to QR code
      */
     logoMarginRatio?: QRCodeLogoMarginRatio;
+
+    /**
+     * If true, the Expensify logo will be displayed
+     */
+    shouldShowExpensifyLogo?: boolean;
+
+    /**
+     * Additional styles to be applied to the QR code
+     */
+    additionalStyles?: StyleProp<ViewStyle>;
+
+    /**
+     * If true, the QR code will be displayed in a different size
+     */
+    isFromDownloadPage?: boolean;
```

Comment:
Let's make this prop `size` instead. Create a `CONST` similar to `CONST.AVATAR_SIZE` and add two size variants.

Reply:
Still, the prop should be `size` and in that style use a `StyleUtils.getQRCodeStyle(size)`